src/api/wallet/composeProvider.ts

```diff
@@ -10,7 +10,7 @@ import { TransactionConfig } from 'web3-core'
 import { numberToHex, hexToNumber } from 'web3-utils'
 import { isWalletConnectProvider, Provider } from './providerUtils'
 import { logDebug } from 'utils'
-import { web3 } from 'api'
+import { web3 } from 'api/gpV1'
```

Comment:
would be cool to have one central web3 api

Reply:
Well, every app can export their own web instance exactly the same way as `gp-v1` does.

But I get your point. Every app will need `web3`. Will change that.